examples/covid19-stat-publisher/processor/processor.bal

```diff
@@ -58,8 +58,10 @@ service "Covid19UpdateDownloader" on secureRemoteServer {
 
     function onFileChange(ftp:WatchEvent event) {
         foreach ftp:FileInfo addedFile in event.addedFiles {
-            string fileName = addedFile.name.clone();
-            log:printInfo("Added file: " + fileName);
+            string fileName = <string>addedFile.name.clone();
```

Comment:
Is there any specific reason to use `clone` here ?

Reply:
Because it is `readonly`.